version.gradle.kts

```diff
@@ -1,5 +1,5 @@
-val stableVersion = "1.16.0-SNAPSHOT"
-val alphaVersion = "1.16.0-alpha-SNAPSHOT"
+val stableVersion = "1.17.0-SNAPSHOT"
```

Comment:
This is where we control the version of the release?

Reply:
yes